spec/controllers/queue_items_controller_spec.rb

```diff
@@ -89,6 +89,15 @@
       expect(QueueItem.count).to eq(0)
     end
 
+    it "normalizes the remaining queue items" do
+      alice = Fabricate(:user)
+      session[:id] = alice.id
+      queue_item1 = Fabricate(:queue_item, user: alice, position: 1)
+      queue_item2 = Fabricate(:queue_item, user: alice, position: 2)
+      delete :destroy, id: queue_item1.id
+      expect(QueueItem.first.position).to eq(1)
```

Comment:
I prefer to be more explicit. Since you have `queue_item2` accessible, I would do:

```ruby
expect(queue_item2.reload.position).to eq(1)
```